input/size.go

```diff
@@ -236,11 +267,19 @@ const (
 	// which will transition an outgoing HTLC to the delay-and-claim state.
 	HtlcTimeoutWeight = 663
 
+	// TaprootHtlcTimeoutWeight is the total weight of the taproot HTLC
+	// timeout transaction.
+	TaprootHtlcTimeoutWeight = 645
```

Comment:
This is unused but happens because it doesn't use a 65 byte local sig in the test (can be seen by spew dumping it)

Reply:
Yeah this is unused, I added it for completion purposes, and also so it can be used in test vectors later. 

We needed the other version pre zero-fee anchors as the second level tx actually had a fee.